elyra/pipeline/component_catalog.py

```diff
@@ -236,7 +236,7 @@ def get_all_components(self, platform: RuntimeProcessorType) -> List[Component]:
         for catalog_name, catalog_components in platform_components.items():
             components.extend(list(catalog_components.values()))
 
-        if not components:
+        if not components and platform.name != "LOCAL":
```

Comment:
Since we have a `RuntimeProcessorType` we should probably compare it directly...
```suggestion
        if not components and platform != RuntimeProcessorType.LOCAL:
```